.changeset/dry-beds-boil.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/themes': patch
+---
+
+Release the latest beta changes of the `@clerk/themes` package
```

Comment:
❓ @anagstef Why did we add a changelog entry for a change made in our `playground` apps?